src/editors/containers/VideoEditor/components/VideoSettingsModal/components/DurationWidget.jsx

```diff
@@ -22,8 +23,23 @@ export const DurationWidget = () => {
     fields: { [hooks.selectorKeys.duration]: hooks.durationWidget },
   });
   const timeKeys = keyStore(duration.formValue);
+
+  const getTotalLabel = (startTime, stopTime) => {
+    if (!stopTime) {
+      if (!startTime) {
+        return 'Full video length'; // TODO internationalize
+      }
+      return `Starts at ${durationFromValue(startTime)}`; // TODO internationalize
+    }
+    const total = stopTime - (startTime || 0);
+    return `Total: ${durationFromValue(total)}`; // TODO internationalize
+  };
+
   return (
-    <CollapsibleFormWidget title="Duration">
+    <CollapsibleFormWidget
+      title="Duration"
+      subtitle={getTotalLabel(duration.formValue.startTime, duration.formValue.stopTime)}
+    >
       <FormGroup size="sm">
         <div>
           <FormControl
```

Comment:
refer to the mock-ups in ticket for the helper text that should be under each of the fields. Also i think that these need to be number inputs versus text inputs.

Reply:
the inputs are text, so they can display '00:00:00' format. duration.js handles the conversion between that and actual numeric values (milliseconds)